### PR TITLE
Updated to 10.11.7

### PR DESCRIPTION
**Pull request by aliasbody**
This PR serves as a base update for the MariaDB client. It updates the SDK from 20.08 to 23.08 and MariaDB from 10.5.11 to 10.11.7 (which is the most up to date LTS version). A future update for 11.x version will come later.

### CODE REVIEW

File: io.dbeaver.DBeaverCommunity.Client.mariadb.json [hunk 1]
```diff
@@ -11,7 +11,10 @@
       "name": "mysql",
       "buildsystem": "cmake",
       "build-options": {
-        "strip": true
+        "strip": true,
+        "build-args": [
+          "--share=network"
```
Comment (hfiguiere) [inline io.dbeaver.DBeaverCommunity.Client.mariadb.json]: build-args are not allowed. the build MUST be offline.

Comment (aliasbody) [reply io.dbeaver.DBeaverCommunity.Client.mariadb.json]: You're right, my bad. I fixed it with a new commit, thank you for your feedback.

**Comment by aliasbody** [comment]
@p1u3o @dac73 Is this good to be merged? Thank you!

**Comment by aliasbody** [comment]
@p1u3o @dac73 Sorry to bother once again, but can this be merged so that the mariadb connection works again? Thanks.

**Comment by dac73** [comment]
Hey man, sure. Why the PAM changes though?

**Comment by aliasbody** [comment]
@dac73 Because otherwise "-DWITH_AUTHENTICATION_PAM=OFF", doesn't have any effect for some reason. It tries to still build it and fails:

`CMake Error at cmake/plugin.cmake:327 (MESSAGE):
  Plugin AUTH_PAM cannot be built
`

**Pull request merged**